lnbits/helpers.py

```diff
@@ -211,3 +216,15 @@ def get_current_extension_name() -> str:
     except:
         ext_name = extension_director_name
     return ext_name
+
+
+def get_push_notification_pubkey():
+    push_notification_key_file = os.path.join(
```

Comment:
is there a reason why this is not inside the database? because if we use postgres that datadir may not even exist